src/core/src/any.cpp

```diff
@@ -63,7 +63,8 @@ void Any::Base::read_to(Base& other) const {
     if (other.is<std::string>()) {
         *static_cast<std::string*>(other.addressof()) = strm.str();
     } else {
-        other.read(strm);
+        if (!strm.str().empty())
+            other.read(strm);
```

Comment:
can we create dedicated ov::Any test for it? what happens if stream is empty?

Reply:
Added test